File: AUGUST-2025/05-08-2025/p3477.go
func numOfUnplacedFruits(fruits []int, baskets []int) int {
    ans := 0
    for i:=0; i<len(fruits); i++{
        for j:=0; j<len(baskets);j++{
            if fruits[i] <= baskets[j]{
                baskets[j] = -1
                break
            }else if j == len(baskets)-1{
                ans++
            }
        }
    }
    return ans
}

File: AUGUST-2025/05-08-2025/p3477.cs
public class Solution {
    public int NumOfUnplacedFruits(int[] fruits, int[] baskets) {
        int n = baskets.Length;
        bool[] vis = new bool[n];
        int cnt = 0;

        foreach(int i in fruits) {
            bool placed = false;
            for(int j = 0; j < baskets.Length; j++) {
                if(!vis[j] && baskets[j] >= i) {
                    vis[j] = true;
                    placed = true;
                    break;
                }
            }
            if(!placed){
                cnt++;
            }
        }
        return cnt;
    }
}